.github/workflows/reusable_canary.yml

```diff
@@ -66,4 +66,8 @@ jobs:
         with:
           project-name: ${{inputs.repository}}-${{inputs.branch}}-${{inputs.os}}-canary
           source-version-override: refs/heads/release
-          hide-cloudwatch-logs: true
\ No newline at end of file
+          hide-cloudwatch-logs: true
+          env-vars-for-codebuild: |
```

Comment:
I think this looks right to me. I was debating whether `TEST_TYPE` should be in here as well, but I don't think so. The intent of the release canary is to be testing the latest PyPI release, so having the test pull from PyPI seems like the correct behavior.